pkg/pyinst/bitmask.spec

```diff
@@ -20,19 +35,12 @@ exe = EXE(pyz,
           debug=False,
           strip=False,
           upx=True,
-          console=False )
+          console=False,
+          icon='../../data/images/mask-icon.ico')
 coll = COLLECT(exe,
-               a.binaries,
-               a.zipfiles,
-               a.datas,
-               strip=False,
-               upx=True,
-               name='bitmask')
-if sys.platform.startswith("darwin"):
```

Comment:
this shouldn't be removed, if we're going to use the same file for both win and osx builds